homeassistant/components/flexpool/config_flow.py

```diff
@@ -0,0 +1,62 @@
+"""Config flow for Flexpool integration."""
+import logging
+import re
+
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+
+from ...const import CONF_ADDRESS
+from .const import DOMAIN  # pylint:disable=unused-import
```

Comment:
No longer needed to disable, pylint fixed this bug.

```suggestion
from .const import DOMAIN
```